util/util.go

```diff
@@ -311,6 +311,7 @@ func NormalizeName(name string) (string, string) {
 		}
 	}
 
+	name = filepath.ToSlash(name)
 	root := GetRootFromPackage(name)
```

Comment:
What do we get from this? The first thing `GetRootFromPackage` does is run `filepath.ToSlash`. `GetRootFromPackage` is sometimes called outside `NormalizeName`.

Is there a problem in the latest code that needs to be addressed?